scripts/ansible/roles/linux/openenclave/tasks/environment-setup-cross-arm.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+- name: Include base setup
+  include_tasks: environment-setup.yml
+
+- name: Mark existing APT repositories as x86/64
+  replace:
+    path: /etc/apt/sources.list
+    regexp: '^deb h'
+    replace: 'deb [arch=amd64,i386] h'
+
+- name: Enable ARM64 architecture
+  command: dpkg --add-architecture arm64
+
+- apt_repository:
+    repo: deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports {{ ansible_distribution_release }} main restricted universe multiverse
+    state: present
+
+- apt_repository:
+    repo: deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports {{ ansible_distribution_release }}-updates main restricted universe multiverse
+    state: present
+
+- apt_repository:
+    repo: deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports {{ ansible_distribution_release }}-backports main restricted universe multiverse
+    state: present
+
+- name: Include vars
+  include_vars:
+    file: "{{ ansible_distribution_release | lower }}-cross-arm.yml"
```

Comment:
I think that you can reuse the [existing variables files](https://github.com/microsoft/openenclave/tree/5b8785e76b65e376f3fa6da206e766f6e6f6ad29/scripts/ansible/roles/linux/openenclave/vars) and just add the `apt_arm_packages` packages list to the `xenial.yml` and `bionic.yml`.